internal/pkg/cli/interfaces.go

```diff
@@ -419,6 +419,11 @@ type envDescriber interface {
 	PublicCIDRBlocks() ([]string, error)
 }
 
+type versionFeatureGetter interface {
```

Comment:
nit: feel free to ignore it - I was just trying to think of a descriptive name for the interface
```suggestion
type versionCompatibilityChecker interface {
```